### PR TITLE
Add "The fifth Open Paradigm: Open Operations" @ EclipseCon 2022

### PR DESCRIPTION
**Pull request by berendt**
Signed-off-by: Christian Berendt <berendt@23technologies.cloud>

### CODE REVIEW

File: _data/conferences_en.yml [hunk 1]
```diff
@@ -3,6 +3,12 @@
   title: "Developing the Open Source Sovereign Cloud Stack"
   details: "Kurt Garloff"
   slides: "20221117-SCS-MercedesFOSS.pdf"
+
+- date: "2022-10-26"
+  event: "EclipseCon 2022"
+  title: "The fifth Open Paradigm: Open Operations"
+  details: "Christian Berendt"
+  slides: "2022-10-EclipseCon-2022.pdf"
```
Comment (fkr) [inline _data/conferences_en.yml:11]: can we name the file equal to all others?

"YYYYMMDD-EclipseCon.pdf"

**Comment by itrich** [comment]
There is a parallel PR: #466

Either we split your talk from #466 or you contribute to the referenced one?

**Comment by berendt** [comment]
> There is a parallel PR: #466
> 
> Either we split your talk from #466 or you contribute to the referenced one?

Feel free to merge this one into #466.

**Comment by itrich** [comment]
Superseded by #466

**Pull request closed without merging**